indico/modules/events/templates/cloner_options.html

```diff
@@ -0,0 +1,42 @@
+{% for cloner in cloners if cloner.is_visible %}
+    {% set available = cloner.is_available -%}
+    <li>
+        <input type="checkbox" name="cloners" id="cloner-{{ cloner.name }}" value="cloner_{{ cloner.name }}"
+               class="js-cloner-option"
+               data-requires="{{ cloner.requires_deep | list | tojson | forceescape }}"
+               data-required-by="{{ cloner.required_by_deep | list | tojson | forceescape }}"
+               {{ 'checked' if available and cloner.is_default }}
+               {{ 'disabled' if not available }}>
+        {#--#}
+        <label for="cloner-{{ cloner.name }}" style="font-weight: normal;">{{- cloner.friendly_name -}}</label>
+    </li>
+
+    <script>
+        (function() {
+            'use strict';
+
+            function _update() {
+                var $this = $(this);
+                // note that the data attributes also contain names of internal cloners which will not
+                // have corresponding checkboxes.  take this into account when changing the code below
```

Comment:
not a typo, double whitespace adds a clearer distinction between sentences

http://english.stackexchange.com/questions/2544/how-many-spaces-should-come-after-a-period-full-stop/2602#2602

> Don't use two spaces **unless you're using a fixed-width font like a typewriter**. If forced to choose only between one space and two, choose one. But if your typesetting system supports it, have a **wider space between sentences**.